pkg/kn/traffic/compute_test.go

```diff
@@ -25,6 +25,7 @@ import (
 )
 
 type trafficTestCase struct {
+	name             string
```

Comment:
that's something I wanted to bring in as subsequent change, thats for thinking through this and improving it.

Reply:
Yeah, I just introduced a simple solution for identifying a failed test scenario so that we can improve this further as we like.

Reply:
Percent. :+1: